apps/model/lib/model/stop.ex

```diff
@@ -112,4 +112,20 @@ defmodule Model.Stop do
         false
     end
   end
+
+  @doc """
+  Indicates whether a stop is "standalone" (is a stopping location and has no parent).
+
+  iex> standalone?(%Stop{location_type: 0, parent_station: nil})
+  true
+
+  iex> standalone?(%Stop{location_type: 0, parent_station: "id"})
+  false
+
+  iex> standalone?(%Stop{location_type: 1, parent_station: nil})
+  false
+  """
+  @spec standalone?(t) :: boolean
+  def standalone?(%__MODULE__{location_type: 0, parent_station: nil}), do: true
+  def standalone?(%__MODULE__{}), do: false
```

Comment:
If we do, I'm not sure where it would be other than here. This is currently the only instance of the word "standalone" in the codebase, for what that's worth; I also searched for other instances of `location_type: 0` and `parent_station: nil`, but nothing relevant came up.

Reply:
I was thinking of `location_type_0_ids_by_parent_ids` which is related, but not quite the same.